site/about.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
head and body are within html, check indentation <--- watch out for this on other css files too